apps/wallet-mobile/src/features/Swap/common/AmountCard/AmountCard.tsx

```diff
@@ -70,7 +70,7 @@ export const AmountCard = ({
               onChangeText={onChange}
               allowFontScaling
               selectionColor={isFocused ? '#242838' : COLORS.TRANSPARENT_BLACK}
-              style={styles.amountInput}
+              style={[styles.amountInput, value === '0' && styles.grayText]}
```

Comment:
I don't think this one's what they want.

Reply:
have gone back and forward regarding it - they want the grey color even when the input is 0 .. at least this is what I understand... can discuss more related to it here https://emurgo.atlassian.net/browse/YOMO-977 @jorbuedo